single.php

```diff
@@ -9,20 +9,62 @@
  * @since    Timber 0.1
  */
 
```

Comment:
Add this to make it easier for our IDE's to link to Timber code.
`use Timber\Timber;`